### ISSUE TITLE
Changing Font in Beamer

### ISSUE THREAD

**Issue by Yusufalbadry** [maumneto/TemplateBeamerUFC#4]
Thank you very much for this template, it was very easy to edit.
I am struggling with changing the font, I made it through changing the language from Brazilian to English, but I want to change the font and I couldnt find the command for the font. I am new to LaTeX and I searched for every command in preamble but I couldn't find any of them related to font except the one for the equations font.

Best regards,
Yusuf

**Comment by maumneto** [comment]
Hi Yusuf!

Thank you for the compliment and for using the template I created. I hope it will be helpful to you. The font I use in the template is the LaTeX standard. However, it is possible to modify it. You can do this locally (specifying the text you want to change) or globally (for the entire template). If you want to do it locally, just use the font command you want to use, for example: \ texttt {}. For more information, please visit this link: https://www.overleaf.com/learn/latex/font_typefaces
Now, if you want to change globally, I suggest you read that link: https://tex.stackexchange.com/questions/79420/changing-font-style-using-beamer.
Basically, you will have to modify the default font for a beamer file.
I hope that this information will be helpful to you. If you have further questions, you can return.

Best Regards,
Maurício.

**Comment by Yusufalbadry** [comment]
I am very sorry for my question, but where should I include these commands ?

\usefonttheme{professionalfonts} % using non standard fonts for beamer
\usefonttheme{serif} % default family is serif
\usepackage{fontspec}
\setmainfont{Liberation Serif}

I tried to add them in document.tex but it is not working

**Comment by maumneto** [comment]
Try to put in ufc_format.cls file. This file has the style commands of this template.
